website/content/en/preview/concepts/node-templates.md

```diff
@@ -34,6 +34,9 @@ spec:
   metadataOptions: { ... }       # optional, configures IMDS for the instance
   blockDeviceMappings: [ ... ]   # optional, configures storage devices for the instance
   detailedMonitoring: "..."      # optional, configures detailed monitoring for the instance
+status:
+  subnets: { ... }               # optional, resolved subnet
+  securityGroups: { ... }        # optional, resolved security groups
```

Comment:
Since the user should never specify these, I don't think we need the optional/required. 

```suggestion
  subnets: { ... }               # resolved subnet
  securityGroups: { ... }        # resolved security groups
```